fuse.go

```diff
@@ -222,35 +222,67 @@ func parseBuf(b []byte, alloc *Allocator, c *Conn) (Request, Response, error) {
 			return nil, nil, fmt.Errorf("fuse: read %d opcode %d but expected %d", len(b), h.opcode, h.len)
 		}
 	}
+	var handleErr error
 	switch h.opcode {
 	case OpInit:
-		return parseInit(b, alloc)
+		return parseInit(b, scope.alloc)
```

Comment:
Are you ok with having the higher level recover as it exists in source's serve.go?